assets/js/theme/custom/add-reviews-to-cart-items.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Add reviews to cart items via GraphQL
+ */
+export default function addReviewsToCartItems(storefrontApiToken) {
+    const cartItems = document.querySelectorAll('.js-item-row')
+    if (cartItems.length > 0) {
+        cartItems.forEach((product) => {
+            const productId = product.getAttribute('data-type-product-id')
+            fetch('/graphql', {
+                method: 'POST',
+                credentials: 'same-origin',
+                headers: {
+                    'Content-Type': 'application/json',
+                    Authorization: `Bearer ${storefrontApiToken}`,
+                },
+                body: JSON.stringify({
+                    query: `
+                        query getProductRating {
+                            site {
+                                product(entityId: ${productId}) {
+                                    reviewSummary {
+                                        numberOfReviews
+                                        summationOfRatings
+                                    }
+                                }
+                            }
+                        }
+                    `,
+                }),
+            })
+                .then((res) => res.json())
+                .then((data) => {
+                    const summationOfRatings = data?.data?.site?.product?.reviewSummary?.summationOfRatings
+                    const numberOfReviews = data?.data?.site?.product?.reviewSummary?.numberOfReviews
+                    const ratingElement = document.querySelector(`[data-type-product-id="${productId}"] .c-cart__item-rating`)
+                    const ratingLink = document.querySelector(`[data-type-product-id="${productId}"] .c-cart__item-rating-link`)
+
+                    ratingLink.innerHTML = `${numberOfReviews} reviews`
+
+                    const filledStarSVG = `
```

Comment:
Not a fan of this solution as you hardcode the SVG inside js. Can't we set a ratings in element's data and based on this value display correct number of stars?